### ISSUE TITLE
Hiding pages.tx_temaplvoilaplus_to with PageTSconfig breaks add page in LIST module

### ISSUE THREAD

**Issue by jpmschuler** [pluspol-interactive/templavoilaplus#256]
8LTS with TV+ 7.3.2-dev and 7.3.1

Steps to reproduce:
1. open list module
2. select a page with subpages from tree
3. in content area click the create page button (see image from official TYPO3 docs, button [1] below)
4. Enter a page title in the new form and click save
5. Exception shows:

```
Oops, an error occurred!
No data structure for field "tx_templavoilaplus_flex" in table "pages" found, no "ds" array configured and given row does not have a field with ds_pointerField name "tx_templavoilaplus_ds".
More information regarding this error might be available online.
```

Reason is probably that the TV+ fields/tabs are missing from this edit form, thus neither a DS nor TO can be selected or rather no TO field is shown and thus the static-DS-automagic isn't run.

![BackendListModule2](https://user-images.githubusercontent.com/12411176/58842894-7d68f380-8670-11e9-9b03-2bc9bbfa8b8a.png)

**Comment by opi99** [comment]
Tried this with TYPO3 8.7.30 and TV+ 7.3.2-dev now and can't reproduce the error.
Does this still happen for you?

**Comment by jpmschuler** [comment]
I can't find a 7.3.2-dev in GitHub or packagist, I tried dev-master 61fdd85 and there the error is reproducible. Trying to debug the problem I found the concrete culprit which is reproducible in dev-master as well as 7.3.1:

For one of our site designs we only have one TO. Thus the idea was to set the TO once for the root of site, remove it from all other pages and hide the field altogether with page.TSconfig in order to remove the amount of fields for editors:
`TCEFORM.pages.tx_templavoilaplus_to.disabled = 1
TCEFORM.pages.tx_templavoilaplus_next_to.disabled = 1`

As soon as I made the fields visible again or used a site with a different TO and config  it was no issue.
